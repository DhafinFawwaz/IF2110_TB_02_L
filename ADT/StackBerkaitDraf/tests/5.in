| Ini isi draf
| Tuan Hak

| Ini juga isi draf
| Tuan Hak