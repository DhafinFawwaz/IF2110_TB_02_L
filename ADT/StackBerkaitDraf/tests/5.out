Panjang Stack Berkait Draf: 2